react-native/debugging/handle-gradle-dependencies-clash.mo.md

```diff
@@ -0,0 +1,116 @@
+# [MO] Handle gradle dependencies versions clashes
+
+## Owner: [Louis Zawadzki](https://github.com/louiszawadzki)
+
+## Prerequisites:
+
+You have 2 dependencies that depend on different versions of the same package, so your application won't build and error with an error like:
+
+```
+com.android.dex.DexException: Multiple dex files define Landroid/support/v7/appcompat/R$anim;
+```
+
+If you're having an issue with Google Maps and Firebase you can have a look at [this article](https://medium.com/@suchydan/how-to-solve-google-play-services-version-collision-in-gradle-dependencies-ef086ae5c75f).
+
+## Steps:
+
+### 1. Know which dependencies is causing the issue *(~2min)*
+
+First, you need to figure out which package is causing the dependency issue.
+
+You can have versions clash without any issue.
+However when one method is present in one version and not in the other there will be an issue.
```

Comment:
For any method, if version A doesn't have the same logic as B, you could still get issues, depending on if you or your dependencies use this method and expect the behaviour of B. Sure, you'll be able to compile, but you still might have some bugs!

Reply:
true, I'll add it here